src/frontend/komponenter/Fagsak/Brevmottaker/LeggTilEndreBrevmottakerModal.tsx

```diff
@@ -22,6 +22,14 @@ const StyledModal = styled(Modal)`
     padding: 1rem;
 `;
 
+const StyledModalHeader = styled(Modal.Header)`
+    padding: 0;
+`;
+
+const StyledModalBody = styled(Modal.Body)`
+    padding: 0;
+`;
+
```

Comment:
Hvorfor ønsker man 0 padding? Foretrekker å bruke komponentene som de kommer fra Aksel med mindre det er en veldig god grunn til å gjøre noe annet

Reply:
Dette var bare for å unngå "mye spacing" - slik at den ble mer lik dagens.
Da har jeg fjernet dette.